components/support/error/macros/src/lib.rs

```diff
@@ -48,7 +48,7 @@ mod argument;
 ///    }
 /// }
 ///
-/// #[handle_error]
+/// #[handle_error(Error)]
```

Comment:
nit: Should we add a note in the doc comment to what `Error` is? maybe even a small comment above the macro invocation in the example that `Error` is the internal error and that it's mandatory to specify